pages/contribute.md

```diff
@@ -6,44 +6,55 @@ main_nav_id: contribute
+If you have a library that could be useful for others, let us know.
 
-### Presentations
+## Contribute to the documentation
 
-If you've given or are giving presentations on Kotlin, let us know. We'll [feature it on the list](http://kotlinlang.org/docs/events.html)
+If you have helpful information for [Kotlin documentation](https://kotlinlang.org/docs/reference/) or have found a documentation issue, feel free to check out [the documentation source code on GitHub](https://github.com/jetbrains/kotlin-web-site),
+commit changes, and send us a Pull Request. Please stay in touch because we may have questions and comments on your changes. Otherwise, we won't be able to incorporate them. 
 
-### Kotlin-x Projects and Community Projects
+If you want to propose changes to a specific page, you can click **Edit Page** right on the page.
+  
+The documentation site is built using Markdown and Jekyll. 
 
-Kotlin ships with a very small runtime and it's the aim to keep it that way. We believe other functionality that is missing from the standard runtime library can be developed
-as Kotlin Contributions under the Kotlin-X umbrella or as individual projects by community members. If you have a library that you think
-others might find useful, let us know. If you want to contribute to any of the existing ones, check them out.
+## Create tutorials or videos
 
-### Translations
+If you've created tutorials or videos for Kotlin, please share them with us. We would be happy to add them to [Awesome Kotlin](https://kotlin.link/) maintained by the community.
 
-You are welcome to translate the Kotlin documentation into your own language and to publish your translation on your Web site.
-However, note that we will not be able to host your translation in this repository and to publish it on kotlinlang.org.
-This site is the official documentation for the language, and we need to be able to ensure that all the information here
-is correct and up-to-date. Doing this with a language that no one on the team speaks is unfortunately not feasible at this time.
+## Translate documentation to other languages
 
-### Spread the word
+You are welcome to translate the Kotlin documentation into your own language and publish your translation on your web site.
+However, we won't be able to host your translation in the main repository and publish it on [kotlinlang.org](https://kotlinlang.org/docs/reference/).
 
-Kotlin is a new language but we have high hopes for it and we need people that believe in it to spread the word! 
+This site is the official documentation for the language, and we ensure that all the information here
+is correct and up to date. Unfortunately, we won't able to review documentation in your own language. 
 
+## Hold events and presentations
+
+If you've given or just plan to give presentations or hold events on Kotlin, let us know. We'll feature them on [the event list](http://kotlinlang.org/docs/events.html)
+
+## Participate in Early Access Preview
```

Comment:
Looks like this won't be visible to very many readers. Should we move it earlier?

Reply:
Moved to the top